cmd/target.go

```diff
@@ -232,14 +226,15 @@ func NewTargetCmd(targetReader TargetReader, targetWriter TargetWriter, configRe
 
 // resolveNameProject resolves name to project
 func resolveNameProject(target TargetInterface, name string) (matches []string) {
-	tmp := KUBECONFIG
-
-	client, err := target.K8SClientToKind(TargetKindGarden)
+	gardenClientset, err := target.GardenerClient()
 	checkError(err)
-	projectLabel := "garden.sapcloud.io/role=project"
-	projectList, err := client.CoreV1().Namespaces().List(metav1.ListOptions{
-		LabelSelector: projectLabel,
-	})
+	if !strings.Contains(name, "*") {
+		project, err := gardenClientset.GardenV1beta1().Projects().Get(name, metav1.GetOptions{})
+		checkError(err)
```

Comment:
```suggestion
if err != nil {
	return []string{}
}
```
so we can preserve also the unit test.